### ISSUE TITLE
Printing Large Amounts of Data Fails

### ISSUE THREAD

**Issue by stdavis** [agrc/deq-enviro#560]
![Image](https://user-images.githubusercontent.com/1326248/263851527-3dbfb399-8acd-4068-bb61-17ba2a09aa70.png)

Load some of these layers into a AGOL web map and see if it has the same issues.

Perhaps we can add a feature result limit.

**Comment by nathankota** [comment]
Attempted to mimick the query in the initial issue comment relatively closely. The map view worked, but the layout view set to pdf output format and the other default settings failed multiple times.



![Image](https://github.com/agrc/deq-enviro/assets/15659642/98438917-9dcc-4164-a7b6-74a6b78c19c0)

**Comment by stdavis** [comment]
I'm not sure that there is a lot that we can do here. I believe that this is a mix of the print proxy request to the AGOL print service timing out and the browser request to print proxy (via the print widget) timing out. I tried increasing the browser request timeout using [`esri.config.request.timeout`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#request) but that didn't seem to have any effect on the request coming from the print widget.

I did fix a few bugs in print proxy that should help with layout templates. I also bumped the node version so maybe that will help with performance? Although, it's not print proxy itself that's timing out.

I think that in the end, folks are just not going to be successful printing 10's of 1000's of features. I'm not sure how valid that use case is anyway.
